File: moneylog_spring/src/main/java/bitc/fullstack503/moneylog_spring/dto/CategoryDTO.java
package bitc.fullstack503.moneylog_spring.dto;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
@Data
public class CategoryDTO
{
  @JsonProperty ("categorySeq")
  private int categorySeq;
  @JsonProperty ("categoryName")
  private String categoryName;
  @JsonProperty ("categoryIsDefault")
  private boolean categoryIsDefault;
  @JsonProperty ("memberId")
  private String memberId;
}

File: moneylog_app/app/src/main/java/bitc/example/app/DTO/CategoryDTO.kt
package bitc.example.app.dto

import com.google.gson.annotations.SerializedName

data class CategoryDTO(
  @SerializedName("categorySeq")
  var categorySeq: Int? = null,
  @SerializedName("categoryName")
  var categoryName: String? = null,
  @SerializedName("categoryIsDefault")
  var categoryIsDefault: Boolean? = null,
  @SerializedName("memberId")
  var memberId: String? = null
)
